pkg/containertools/factory_podman.go

```diff
@@ -0,0 +1,33 @@
+package containertools
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+type PodmanCommandFactory struct{}
+
+func (p *PodmanCommandFactory) BuildCommand(o BuildOptions) (*exec.Cmd, error) {
+	args := []string{"build"}
+
+	if o.format != "" {
```

Comment:
Dumb thought: do we ever want to consider just passing arbitrary command arguments here rather than being explicit with flags? It might make for a simpler cli experience. Especially if we make the default for build fully daemonless

The counter is that we're on the hook for making all of these arbitrary flags work? 🤷

Reply:
I think you could make a good argument for that, but I lean towards letting users directly invoke podman/whatever if they are in a circumstance that calls for it.